runtime/wasm/src/host.rs

```diff
@@ -154,6 +154,10 @@ impl RuntimeHost {
                     .event_handlers
                     .iter()
                     .map(|event_handler| {
+                        debug!(logger, "Prepare subscription";
+                            "event" => &event_handler.event,
```

Comment:
Small misalignment here.

Reply:
I'm not clear on the proper alignment for the logging macros; I looked at our codebase for reference and see the second line indented differently in a few spots.

Reply:
Yeah, `rustfmt` doesn't handle them properly. The goal should always be this:
```
debug!(logger, "Prepare subscription";
       "event" => &event_handler.event,
       ...)
```
So: align the `"key" => value` pairs with the other arguments to the macro.